optuna/samplers/tpe/sampler.py

```diff
@@ -447,3 +453,45 @@ def _log_normal_cdf(cls, x, mu, sigma):
         numerator = np.maximum(np.sqrt(2) * sigma, EPS)
         z = denominator / numerator
         return .5 + .5 * scipy.special.erf(z)
+
+    @staticmethod
+    def hyperopt_parameters():
```

Comment:
In the future, we may want to add other reference settings. E.g., parameters in the following papers:

- Algorithms for Hyper-Parameter Optimization
  - https://papers.nips.cc/paper/4443-algorithms-for-hyper-parameter-optimization.pdf
- Making a Science of Model Search: Hyperparameter Optimization in Hundreds of Dimensions for Vision Architectures
  - http://proceedings.mlr.press/v28/bergstra13.pdf